templates/repo/issue/view_content/pull.tmpl

```diff
@@ -321,8 +321,25 @@
 					{{if .AllowMerge}}
 						{{$prUnit := .Repository.MustGetUnit $.UnitTypePullRequests}}
 						{{$approvers := .Issue.PullRequest.GetApprovers}}
+						{{/* We build a custom variable here to not clutter the template with checks if .LatestCommitStatus exists. */}}
+						{{$statusChecksPending := false}}
+						{{if .LatestCommitStatus}}
+							{{$statusChecksPending = eq .LatestCommitStatus.State "pending"}}
+						{{end}}
```

Comment:
I am almost tempted to suggest
```suggestion
						{{$statusChecksPending := true}}
						{{if .LatestCommitStatus}}
							{{$statusChecksPending = not (eq .LatestCommitStatus.State "success")}}
						{{end}}
```
instead as while it is somewhat understandable that such a check must exist for this to be displayed,
it is at the same time somewhat unintuitive to only add it in that case...

I'm undecided on this, what do you think about it?